src/services/dashboard.js

```diff
@@ -206,7 +163,7 @@ export async function queryIterationSamples(params) {
                   aggs: {
                     title: {
                       terms: {
-                        field: 'sample.measurement_title',
+                        field: 'sample.measurement_title_kw',
```

Comment:
Base on https://github.com/distributed-system-analysis/pbench/pull/2053, shouldn't this be `'sample.measurement_title.measurement_title_kw'`?

Reply:
Oddly, ES7 didn't complain about this query, so maybe it really does work as I thought I'd seen last year. However, as you'd pointed out in the other PR, current ES7 documentation references it as a scoped sub-field name, so I'm changing this. I thought the name was pretty awkward, anyway. ;-)